src/bin/uhyve.rs

```diff
@@ -73,6 +73,11 @@ struct Args {
 	#[cfg(target_os = "linux")]
 	gdb_port: Option<u16>,
 
+	/// Environment variables of the guest as env=value paths. `-e host` passes all variables of the parent process to the kernel.
+	/// Example: --env_vars ASDF=jlk -e TERM=uhyveterm2000
```

Comment:
This would hide the example in the short help and show it more nicely in the long help:

```suggestion
	/// Environment variables of the guest as env=value paths. `-e host` passes all variables of the parent process to the kernel.
	///
	/// Example: --env_vars ASDF=jlk -e TERM=uhyveterm2000
```

Reply:
Right!